packages/dds/tree/src/test/shared-tree/treeCheckout.spec.ts

```diff
@@ -707,7 +707,8 @@ describe("sharedTreeView", () => {
 
 		assert.equal(checkout2Revertibles.undoStack.length, 1);
 		assert.equal(checkout2Revertibles.redoStack.length, 1);
-		assert.deepEqual(checkout2.getRemovedRoots().length, 2);
+		// trunk trimming causes a revertible to be disposed
```

Comment:
No revertible is being disposed here, right? That would be a bug if it were the case.
```suggestion
		// trunk trimming causes a removed root to be garbage collected
```